sma-webconnect.js

```diff
@@ -33,6 +33,28 @@ module.exports = function (RED) {
     }
   };
 
+  const sb = {
+    "id": "1",
+    "values": {
+      "6100_00464800": {
```

Comment:
Is there a reason, why you're using `6100_00464800` instead of `6100_0046E500`? Is that one not supported on the Sunny Boy? My Sunny Tripower supports both of them, but only the latter provides a value during the night.

The same question for the other values:
power: `6100_0046C200` vs. `6100_40263F00`
current: `6100_40465300` vs. `6100_40465300`

Reply:
6100_0046E500 and the other values that your Sunny Tripower supports are not supported by my Sunny Boy.
The Sunny Boy stops reporting 6100_00464800 approximately five minutes after the sun sets and appears to go into a stand by mode. I do not have an SMA Energy Meter as part of my system. Do you have one? Could it be providing the measurements via the inverter?

Reply:
Okay, that's good to know. And yes, I do have a "Sunny Home Manager 2.0", and it may be very possible that those values are provided by it.

So as the Sunny Tripower supports both of those ID's I'm wondering if we should find a different key name for the values you have to use for the Sunny Boy vs the existing ones.

The web interface of my tripower isn't the biggest help here.

The values you're using are found here:
AC Side -> Grid measurements -> Phase currents

The one's I'm using from my Tripower are here:
AC Side -> Measured values -> Grid measurements -> Phase currents

So my first instinct would be to prefix the existing ones with "grid_", but as that would be a breaking change, that's not an option 😖 So I'm almost inclined, to keep it as it is.

Reply:
Are you able to do a test to see if when your "Sunny Home Manager 2.0" ethernet is unplugged, the "AC Side -> Measured values -> Grid measurements -> Phase currents, etc" disappear?
If the measurements do disappear, the current config for the Sunny Tripower will only be suitable for those who have a "Sunny Home Manager 2.0" and it might be worthwhile doing one of the following:

- Include both `6100_0046E500` and `6100_00464800` in one config and the user can pick which one works.
- A tick box to enable "Sunny Home Manager 2.0" support
- Have separate configs to allows for a `sunny_tripower` and a `sunny_tripower_home_manager`

However it might be hard to do these without breaking changes.
What are your thoughts on this?

Reply:
Your assumption is correct, the values under that section all get dashed out ("------") in the web interface after about 10 seconds after disconnecting the Home Manager. At least at nighttime, but I'd like to test it again during daytime tomorrow.

Another option, to stay compatible, would be to just rename the display name from "SB TRIPOWER 8.0/10.0" to "SB TRIPOWER 8.0/10.0 with Home Manager 2.0" and introduce a new preset `sunny_tripower_standalone` with the old display name "SB TRIPOWER 8.0/10.0". Not the prettiest solution from the codes perspective, but without breaking stuff.

Reply:
I've tested it again today during the day, the behavior is the same.